pom.xml

```
@@ -177,7 +177,7 @@
         <components-font-awesome.version>4.7.0</components-font-awesome.version>
         <countryconverter.version>0.1.8</countryconverter.version>
         <cxf.version>3.2.12</cxf.version>
-        <ddf-admin-ui.version>2.25.0-SNAPSHOT</ddf-admin-ui.version>
+        <ddf-admin-ui.version>2.25</ddf-admin-ui.version>
```

Should this be 2.25.0?
```suggestion
        <ddf-admin-ui.version>2.25.0</ddf-admin-ui.version>
```

---

yes, not sure how I managed that one. Thanks